tests/data/SPECS/dynamic.spec

```diff
@@ -0,0 +1,39 @@
+Summary: dynamic hello -- hello, world rpm
+Name: dynamic
+Version: 1.0
+Release: 1
+Group: Utilities
+License: GPL
+Distribution: RPM test suite.
+URL: http://rpm.org
+BuildArch:	noarch
+
+%description
+Simple rpm demonstration.
+
+%prep
+%setup -q -T -c
+
+%build
+echo "Q: Why?\nA: Because we can!" > FAQ
+
+%install
+mkdir -p $RPM_BUILD_ROOT/usr/local/bin
+echo " " > $RPM_BUILD_ROOT/usr/local/bin/hello
+
+
+echo "%package docs" >> %{specpartsdir}/docs.specpart
+echo "Summary: Documentation for dynamic spec" >> %{specpartsdir}/docs.specpart
```

Comment:
Nitpick, but I'd add a comment saying that a `%description` is missing here which makes the build fail (otherwise it's not that much apparent from the looks of it).

Reply:
I suppose this comment was meant for the dynamic-fail.spec instead. But yeah, just having a fail case here depending on a macro would make it more obvious too.

Reply:
Oh, yup, I confused file :)

Reply:
With both spec files now merged the section is marked by a macro query.